pkg/capabilities/consensus/ocr3/consensus_builders_generated.go

```diff
@@ -0,0 +1,194 @@
+// Code generated by github.com/smartcontractkit/chainlink-common/pkg/capabilities/cli, DO NOT EDIT.
+
+package ocr3
+
+import (
+    "github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+    "github.com/smartcontractkit/chainlink-common/pkg/workflows"
+    streams "github.com/smartcontractkit/chainlink-common/pkg/capabilities/triggers/streams"
+)
+
+func (cfg ConsensusConfig) New(w *workflows.Workflow,ref string, input ConsensusInput)ConsensusCap {
+    def := workflows.StepDefinition{
+       ID: "offchain_reporting@1.0.0",Ref: ref,
+       Inputs: workflows.StepInputs{
+           Mapping: map[string]any{
+               "observations": input.Observations.Ref(),
+           },
+       },
+       Config: map[string]any{
+           "aggregation_config": cfg.AggregationConfig,
+           "aggregation_method": cfg.AggregationMethod,
+           "encoder": cfg.Encoder,
+           "encoder_config": cfg.EncoderConfig,
+           "report_id": cfg.ReportId,
+       },
+       CapabilityType: capabilities.CapabilityTypeConsensus,
+   }
+    step := workflows.Step[SignedReport]{Definition: def}
+     raw := workflows.AddStep(w, step)
+    return &consensus{CapDefinition: raw}
```

Comment:
Given a method on `Step`, these can be reduced:
```suggestion
    return &consensus{CapDefinition: step.AddTo(w)}
```